source/components/organisms/SummaryList/SummaryList.tsx

```diff
@@ -40,6 +40,12 @@ const SumContainer = styled.View<{ colorSchema: string }>`
   background: ${(props) =>
     props.theme.colors.complementary[props.colorSchema][3]};
 `;
+
+const SummarySpacer = styled.View`
+  padding-bottom: 16px;
+  width: 100%;
+`;
+
```

Comment:
How would it look like if you added a bottom border after each child? Except the last child?

Reply:
Bra inspel. Verifiera detta med Maria.